pom.xml

```diff
@@ -80,7 +80,7 @@
         <version.microservice.accumulo-utils>1.3</version.microservice.accumulo-utils>
         <version.microservice.audit-api>1.4</version.microservice.audit-api>
         <version.microservice.authorization-api>1.4</version.microservice.authorization-api>
-        <version.microservice.base-rest-responses>1.2</version.microservice.base-rest-responses>
```

Comment:
I might be confused. There's already a 1.2 base-rest-responses jar deployed to github, so I'm not understanding how this PR can be dependent on a 1.2-SNAPSHOT. This PR should depend on a new 1.3 release. Likewise, your current base-rest-responses PR should be targeting 1.3-SNAPSHOT at the moment

Right?

Reply:
@keith-ratcliffe ,  @ivakegg  wanted me to do it this way I am pretty sure.  I had to change some code in that project  (base-rest-responses) for this pull request.  That is to say that there is an associated pull request in that project which I mention above.

Reply:
I will fix the version when I merge this in.